src/modules/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import { QuickLinks } from '../QuickLinks';
+import { WhatsNew } from '../WhatsNew';
+import styles from './SideBar.module.css';
+
+const cn = classNames.bind(styles);
+
+export function SideBar() {
```

Comment:
I tnink we should add className prop for case when we have to control some styles from outside.